src/components/structures/MatrixChat.tsx

```diff
@@ -471,6 +477,35 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         if (this.accountPasswordTimer !== null) clearTimeout(this.accountPasswordTimer);
     }
 
+    private onKeyDown = (event: KeyboardEvent): void => {
```

Comment:
you can open the devtools in many more ways, e.g Inspect Element, browser's View menu etc

![image](https://user-images.githubusercontent.com/2403652/150762660-10a46f4d-c167-4259-a63b-e957bc33144c.png)

So I don't think this is fool-proof enough and a good scammer could just instruct them to use one of those instead of the keyboard shortcut to avoid this warning

Reply:
Args, I am not idiot... Though I have no idea how to detect the console in another way...

Reply:
> Could do it on viewport resize (unreliable way of detecting console state) but that's still far from ideal, maybe make a localStorage flag which enables debug log and otherwise hide non-critical/bug console.logs.

https://github.com/vector-im/element-web/issues/2803#issuecomment-298773761

Reply:
Not, only I am an idiot but I am also blind :man_facepalming: 

Thanks, will have a look